projects/mysadna/src/app/components/project-details/project-details.component.ts

```diff
@@ -28,17 +27,17 @@ export class ProjectDetailsComponent implements OnInit {
 
   getProject(): void {
     const projectId: number = +this.route.snapshot.paramMap.get('projectId');
-    this.dataService.getProject(projectId)
-      .subscribe(project => {
-        this.project = project;
+    this.firebaseService.getProjectList()
+      .subscribe(projectList => {
```

Comment:
You need to remember about active subscriptions, which can be called several times.
I'll tell you later how to unsubscribe on destroy. But if you're interested, feel free to google it instead.

Reply:
I don't know about them enough, yet.

Reply:
:+1: